src/ameba/config.cr

```diff
@@ -267,6 +266,16 @@ class Ameba::Config
     end
 
     macro included
+      GROUP_SEVERITY = {
+        Lint:        Ameba::Severity::Warning,
+        Metrics:     Ameba::Severity::Warning,
+        Performance: Ameba::Severity::Warning,
+      }
```

Comment:
this is a breaking change that i'm not sure we would like to introduce. The client could rely on `--fail-level Warning` flag and which executed only Lint rules, but now it executes Metrics and Performance in addition

Reply:
That's correct, and I believe we should do this BC since these two groups are more serious than a `convention` severity.